lib/index.js

```diff
@@ -15,6 +15,7 @@ import {a} from './a.js'
 import {an} from './an.js'
 
 const ruleId = 'retext-indefinite-article:retext-indefinite-article'
+const url = 'https://github.com/retextjs/retext-indefinite-article#readme'
```

Comment:
Is this something we could read or construct from the package.json file?

Reply:
- That would read the whole `package.json` in browsers
- Reading JSON would work in CJS, but currently needs `fs` in Node in ESM

Reply:
see related discussion in https://github.com/retextjs/retext-contractions/pull/8#discussion_r771552048